### ISSUE TITLE
[ADD] Show Sensors on LCD option

### ISSUE THREAD

**Issue by juanpintom** [juanpintom/Souliss_ESP_Examples#22]
With an option at webinterface we can select an option to wire from pins 4 , 5 to an I2C LCD and show sensors on it.

**Comment by marksev1** [comment]
Yes +1, would be nice to show sensor values on a 16x2 (or 20x4) i2c lcd connected to ESP.
